tests/unit/test_network_connections.py

```diff
@@ -376,6 +376,12 @@ def test_validate_int(self):
         v = network_lsr.argument_validator.ArgValidatorNum("state", required=True)
         self.assertValidationError(v, None)
 
+        v = network_lsr.argument_validator.ArgValidatorNum(
+            "rx", val_min=0, val_max=0xFFFFFFFF, default_value=None
+        )
```

Comment:
Is the new validator required here? Can you reuse the previous one?

Reply:
New validator is not required, I reused the previous validator, thanks.